<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="VTIPL-H-PC249-L" failures="0" tests="2" name="com.mycompany.automation.AppTest" time="19.499" errors="1" timestamp="2024-05-08T08:21:58 IST" skipped="0">
  <testcase classname="com.mycompany.automation.AppTest" name="sampleText" time="19.323"/>
  <system-out/>
  <testcase classname="com.mycompany.automation.AppTest" name="shouldAnswerWithTrue" time="0.176">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.addScreenCaptureFromPath(String)&quot; because the return value of &quot;com.cucumber.listener.Reporter.getCurrentStep()&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.addScreenCaptureFromPath(String)" because the return value of "com.cucumber.listener.Reporter.getCurrentStep()" is null
at com.cucumber.listener.Reporter.addScreenCaptureFromPath(Reporter.java:84)
at com.mycompany.automation.SeleniumTestHelper.assertEquals(SeleniumTestHelper.java:52)
at com.mycompany.automation.AppTest.shouldAnswerWithTrue(AppTest.java:28)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- shouldAnswerWithTrue -->
  <system-out/>
</testsuite> <!-- com.mycompany.automation.AppTest -->
